pallets/funding/src/benchmarking.rs

```diff
@@ -458,24 +458,98 @@ mod benchmarks {
 		let project_id = inst.create_new_project(project_metadata.clone(), issuer.clone());
-		let original_metadata_hash = project_metadata.offchain_information_hash.unwrap();
-		let edited_metadata_hash: H256 = hashed(EDITED_METADATA);
+
+		let project_metadata = ProjectMetadataOf::<T> {
+			token_information: CurrencyMetadata {
+				name: BoundedVec::try_from("Contribution Token TEST v2".as_bytes().to_vec()).unwrap(),
+				symbol: BoundedVec::try_from("CTESTv2".as_bytes().to_vec()).unwrap(),
+				decimals: ASSET_DECIMALS + 2,
+			},
+			mainnet_token_max_supply: BalanceOf::<T>::try_from(100_000_000_000_0_000_000_000u128)
+				.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+			total_allocation_size: BalanceOf::<T>::try_from(200_000_000_0_000_000_000u128)
+				.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+			auction_round_allocation_percentage: Percent::from_percent(30u8),
+			minimum_price: 11u128.into(),
+			bidding_ticket_sizes: BiddingTicketSizes {
+				professional: TicketSize::new(
+					Some(
+						BalanceOf::<T>::try_from(5000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+					Some(
+						BalanceOf::<T>::try_from(10_000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+				),
+				institutional: TicketSize::new(
+					Some(
+						BalanceOf::<T>::try_from(5000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+					Some(
+						BalanceOf::<T>::try_from(10_000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+				),
+				phantom: Default::default(),
+			},
+			contributing_ticket_sizes: ContributingTicketSizes {
+				retail: TicketSize::new(
+					Some(
+						BalanceOf::<T>::try_from(5000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+					Some(
+						BalanceOf::<T>::try_from(10_000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+				),
+				professional: TicketSize::new(
+					Some(
+						BalanceOf::<T>::try_from(5000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+					Some(
+						BalanceOf::<T>::try_from(10_000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+				),
+				institutional: TicketSize::new(
+					Some(
+						BalanceOf::<T>::try_from(5000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+					Some(
+						BalanceOf::<T>::try_from(10_000 * US_DOLLAR)
+							.unwrap_or_else(|_| panic!("Failed to create BalanceOf")),
+					),
+				),
+				phantom: Default::default(),
+			},
+			participation_currencies: vec![AcceptedFundingAsset::USDT, AcceptedFundingAsset::USDC].try_into().unwrap(),
+			funding_destination_account: issuer_funding.clone().clone(),
+			offchain_information_hash: Some(hashed(format!("{}-{}", METADATA, 69)).into()),
+		};
```

Comment:
I think we can again use the initial project metadata to benchmark this extrinsic? So we don't have to have this 80 line monster haha

Reply:
Initial one has a lot of None fields. This is benchmarking changing it to one that is completely different and bigger when encoded, so probably the proof size would change